.travis.yml

```diff
@@ -3,6 +3,7 @@ sudo: false
 dist: trusty
 language: erlang
 otp_release:
+  - 24.0
```

Comment:
A current problem is that Travis-CI is no good for us anymore. We need to move to something new, probably Github Actions.